src/ProjectService.Business/Commands/Department/EditProjectDepartmentCommand.cs

```diff
@@ -38,10 +38,15 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(EditProjectDepartm
         return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.Forbidden);
       }
 
-      if (!await _departmentRepository.EditAsync(request.ProjectId, request.DepartmentId))
+      bool isEdit = await _departmentRepository.EditAsync(request.ProjectId, request.DepartmentId);
```

Comment:
From performance perspective I'd removed await from isEdit assignment and switched positions of conditions(I mean isEdit and Id checks) and added await inside if statement, because if first condition doesn't match the 'and condition' the second will not be checked

Reply:
done